src/main/java/mate/academy/Main.java

```diff
@@ -8,14 +8,13 @@
 import mate.academy.service.OrderServiceImpl;
 
 public class Main {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws AuthenticationException {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
+            throw new AuthenticationException("Username or password are incorrect");
```

Comment:
It is not your job to throw exceptions. You should simply replace e.printStackTrace(); with a log